src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,3 +1,36 @@
 import React from 'react';
+import { Post } from '../../types/post';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => <>Put the post here</>;
+type Props = {
+  post: Post;
+};
+
+const noComments: JSX.Element = (
+  <>
+    <hr />
+    <b data-cy="NoCommentsMessage">No comments yet</b>
+  </>
+);
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const { user, title, body, comments } = post;
+  const postetByText = `Posted by ${(<UserInfo user={user} />)}`;
```

Comment:
don't use template literals if your component returns jsx element and instead use conditional rendering for UserInfo so it only appears if user exists on post